Library/Homebrew/resource.rb

```diff
@@ -115,6 +115,8 @@ def files(*files)
   def fetch(verify_download_integrity: true)
     HOMEBREW_CACHE.mkpath
 
+    fetch_patches
+
     begin
       downloader.fetch
     rescue ErrorDuringExecution, CurlDownloadStrategyError => e
```

Comment:
We need to make sure the formula installer never invokes `Formula.downloader.fetch` directly (I believe it does currently if I recall correctly) as doing that bypasses this override.

Reply:
Done!